sections/Experiment.tex

```diff
@@ -1,12 +1,96 @@
 \section{Experiments}\label{sec:experiment}
 
+The goal of our experiment is to test the quality of various \gls{ir} models and combinations of \gls{ir} models.
+
+We test various combinations of the following baselines:
+\begin{itemize}
+	\item \gls{lm}
+	\item \gls{lda}
+	\item \gls{bm25}
+	\item \gls{tf-idf}
+	\item \gls{pr}
+	\item \gls{ppr}
+\end{itemize}
+\todo[inline]{clarify the difference between lda and this ir method based on lda. Maybe calling it reverse lda or something?}
+\todo[inline]{add a small description of each baseline, especially the ones not described earlier in the paper}
+
+Each baseline will be given a query and based on that query produce a document-value vector with one value for each document indicating the relevance of the document based on the query.
+Our evaluation results are based on the ranking order of these values.
+When multiple baselines are combined, their document-value vector are normalized and then added or multiplied together element-wise.
+However, in this experiment we will not only test the ability to retrieve a specfic document based on a query related to the document, but also the ability to retrieve relevant documents within a specific topic, generated by our topic model.
+
+For each baseline we evaluate the quality of it's \gls{ir} for both queries generated based on documents and queries generated based on topics.
+For topic queries, all documents with values higher than the mean value in the topic's vector in the document-topic matrix $\theta_t$ are considered correct retrivals.
+This also means that while document queries always only have one ground truth positive, topic queries can have several thousands (~1.000-10.000).
+This makes them hard to compare on a single baseline, however it is still possible to compare these two results between multiple baselines.
+
+For evaluation metrics/measures we use \gls{map}. We also use $p @ n$ for the topic queries. However since document queries only contain one ground truth positive, $p @ n$ doesn't make much sense to show. Instead we show average ranking of the relevant document for document queries.
```

Comment:
```suggestion
For evaluation metrics/measures we use \gls{map}. We also use $p @ n$ for the topic queries.
However since document queries only contain one ground truth positive, $p @ n$ doesn't make much sense to show. 
Instead, we show the average ranking of the relevant document for document queries.
```

Reply:
Der skal også kort beskrives hvad p@n står for og hvordan det udregnes, hvis det ikke bliver gjort et andet sted.